tests/ops/integration_tests/saas/test_oracle_responsys_task.py

```diff
@@ -59,6 +59,34 @@ async def test_access_request_by_phone_number(
             == oracle_responsys_identity_phone_number[1:]
         )
 
+    @pytest.mark.parametrize(
+        "dsr_version",
+        ["use_dsr_3_0", "use_dsr_2_0"],
+    )
+    async def test_access_request_by_email_and_extra_identities(
+        self,
+        dsr_version,
+        request,
+        oracle_responsys_runner: ConnectorRunner,
+        policy,
+        oracle_responsys_identity_email: str,
+    ):
+        request.getfixturevalue(dsr_version)  # REQUIRED to test both DSR 3.0 and 2.0
+
+        access_results = await oracle_responsys_runner.access_request(
+            access_policy=policy,
+            identities={
+                "email": oracle_responsys_identity_email,
+                "ga_client_id": "extra_value",
+            },
+        )
+        assert (
```

Comment:
Should we also assert the `ga_client_id` value?

Reply:
For now I think its ok to just do positive assertions (as opposed to which identities are _not_ returned in the access_results). The main thing we want here is to ensure we don't throw an identity err before accessing data when extra identities are provided.